pkg/tar/tar.go

```diff
@@ -38,10 +38,10 @@ var ErrNotSupportedPlatform = errors.New("platform and architecture is not suppo
 // root of the tar file and should be cleaned (for example using filepath.Clean)
 type PathWhitelistMap map[string]struct{}
 
-// extractTar extracts a tarball (from a tar.Reader) into the root directory
-// if pwl is not nil, only the paths in the map are extracted.
-// If overwrite is true, existing files will be overwritten.
-func extractTar(tr *tar.Reader, overwrite bool, pwl PathWhitelistMap, uidRange *uid.UidRange) error {
+// ExtractTarInsecure extracts a tarball (from a tar.Reader) into the root
+// directory if pwl is not nil, only the paths in the map are extracted. If
+// overwrite is true, existing files will be overwritten.
+func ExtractTarInsecure(tr *tar.Reader, overwrite bool, pwl PathWhitelistMap, uidRange *uid.UidRange) error {
```

Comment:
`pkg/aci/render.go` calls `ExtractTar` in the `pkg/tar/chroot.go` file, so I don't think it can easily be moved out of `pkg/tar`.